buildSrc/src/main/groovy/org/elasticsearch/gradle/vagrant/VagrantTestPlugin.groovy

```diff
@@ -386,44 +443,57 @@ class VagrantTestPlugin implements Plugin<Project> {
                 environment vagrantEnvVars
                 dependsOn up
                 finalizedBy halt
-                commandLine 'vagrant', 'ssh', box, '--command',
-                        "set -o pipefail && echo 'Hello from ${project.path}' | sed -ue 's/^/    ${box}: /'"
             }
             vagrantSmokeTest.dependsOn(smoke)
-
-            Task batsPackagingTest = project.tasks.create("vagrant${boxTask}#batsPackagingTest", BatsOverVagrantTask) {
-                remoteCommand BATS_TEST_COMMAND
-                boxName box
-                environmentVars vagrantEnvVars
-                dependsOn up, setupPackagingTest
-                finalizedBy halt
+            if (box in LINUX_BOXES) {
```

Comment:
Please use a `LINUX_BOXES.contains(box)` instead of groovy's in operator, so we stay as close to java as possible (we would like to eventually switch buildSrc to all java, for performance reasons, so keeping the code as java like as possible will help with that transition). Ditto below for the other similar conditions.